src/Build/BackEnd/Components/Logging/ILoggingService.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Reflection;
```

Comment:
Is it needed?

Reply:
That is relict of using `AssemblyName`. For some reason, my VS doesn't detect unused usings...